src/lib/DriverMessage.ml

```diff
@@ -0,0 +1,6 @@
+type message =
```

Comment:
It is required to pull this out into it's own module to avoid a dependency cycle between the Driver and Conversions